### PR TITLE
Fix chart_best_practices links in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by mhumeSF**
#### What this PR does / why we need it:
Fix chart_best_practices links in CONTRIBUTING.md

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x ] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -104,9 +104,9 @@ This repository is used by Chart developers for maintaining the official charts
 * Provide a secure default configuration
 * Do not leverage alpha features of Kubernetes
 * Includes a [NOTES.txt](https://github.com/helm/helm/blob/master/docs/charts.md#chart-license-readme-and-notes) explaining how to use the application after install
-* Follows [best practices](https://github.com/helm/helm/tree/master/docs/chart_best_practices)
-  (especially for [labels](https://github.com/helm/helm/blob/master/docs/chart_best_practices/labels.md)
-  and [values](https://github.com/helm/helm/blob/master/docs/chart_best_practices/values.md))
+* Follows [best practices](https://github.com/helm/helm-www/tree/master/content/en/docs/chart_best_practices)
+  (especially for [labels](https://github.com/helm/helm-www/blob/master/content/en/docs/chart_best_practices/labels.md)
+  and [values](https://github.com/helm/helm-www/blob/master/content/en/docs/chart_best_practices/values.md))
```
Comment (unguiculus) [inline CONTRIBUTING.md]: I think it would be better to link to the rendered docs site at https://helm.sh/docs instead.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -104,9 +104,9 @@ This repository is used by Chart developers for maintaining the official charts
 * Provide a secure default configuration
 * Do not leverage alpha features of Kubernetes
 * Includes a [NOTES.txt](https://github.com/helm/helm/blob/master/docs/charts.md#chart-license-readme-and-notes) explaining how to use the application after install
```
Comment (unguiculus) [inline CONTRIBUTING.md]: Could you also fix this link, please?

**Review by unguiculus — CHANGES_REQUESTED** [review]
Thanks for catching this.

File: CONTRIBUTING.md (hunk 1 above)
Comment (mhumeSF) [reply CONTRIBUTING.md]: Doh, this makes sense

**Review by unguiculus — APPROVED** [review]

**Pull request merged**